docs/bigquery-usage.rst

```diff
@@ -281,7 +281,7 @@ Run a query which can be expected to complete within bounded time:
    >>> from gcloud import bigquery
    >>> client = bigquery.Client()
    >>> query = """\
-   SELECT count(*) AS age_count FROM dataset_name.person_ages
+   SELECT count(*) AS age_count FROM [project-name:dataset_name.person_ages]
```

Comment:
Isn't `project-name` unnecessary? Per the [BQ docs](https://cloud.google.com/bigquery/query-reference#from), the client's `project` is used, if not supplied.